src/visitpy/common/visitmodule.C

```diff
@@ -12155,9 +12158,10 @@ visit_PythonQuery(PyObject *self, PyObject *args, PyObject *kwargs)
             return NULL;
         }
 
-        char *str_val = PyString_AsString(res);
-        args_pickled = std::string(str_val);
-        PyString_AsString_Cleanup(str_val);
```

Comment:
This pattern is used to make sure deal with unicode issues, it's a common pattern i used in the port, so it would be good to know what failed.

That said,  you have a fix so I don't want to hold things up. Can we add a ticket to examine this in the future? (We should be able to uses std::strings  with the proper choreography )

Reply:
I submitted #5415.